### PR TITLE
Make BackupCommand use config-based permission level

### LINKED ISSUE #526: Backup the minecolonies.dat [$10 awarded]
It would be nice if the minecolonnies.dat was backup automatically at each start of the server/world.

- Backup the dat file as minecolonnies-YYYY-MM-DD.dat and continue to use minecolonnies.dat 
- Do it when the server start
- add a command to do the back so server admin could schedule an automatic backup even when the server is not restarted

[Bountysource](https://www.bountysource.com/issues/41473573-backup-the-minecolonnies-dat?utm_source=41473573&utm_medium=shield&utm_campaign=ISSUE_BADGE)

<bountysource-plugin>

---
The **[$10 bounty](https://www.bountysource.com/issues/41473573-backup-the-minecolonies-dat?utm_campaign=plugin&utm_content=tracker%2F43175463&utm_medium=issues&utm_source=github)** on this issue has been claimed at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F43175463&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by asiekierka**
Closes #526 (again)

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/commands/BackupCommand.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.minecolonies.coremod.commands;
 
 import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.configuration.Configurations;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/commands/BackupCommand.java]: MINOR Remove this unused import 'com.minecolonies.coremod.configuration.Configurations'. [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by marvin-bitterlich** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/635/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/635/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**